upi/vsphere/README.md

```diff
@@ -33,7 +33,7 @@ The bootstrap ignition config must be placed in a location that will be accessib
 5. Ensure that you have you AWS profile set and a region specified. The installation will use create AWS route53 resources for routing to the OpenShift cluster.
 
 6. Run `terraform apply -auto-approve`.
-This will create the OpenShift cluster
+This will reserve IP addresses for the VMs.
 
 7. Run `openshift-install upi bootstrap-complete`. Wait for the bootstrapping to complete.
```

Comment:
I had to run the new command `wait-for` to continue.
```
./openshift-install wait-for bootstrap-complete
INFO Waiting up to 30m0s for the Kubernetes API at https://api.dphillip.devcluster.openshift.com:6443... 
INFO API v1.13.4+c68d654 up                       
INFO Waiting up to 30m0s for the bootstrap-complete event... 
```

Reply:
Yep. I'll have an update for this primed for a subsequent PR.